.github/workflows/angular.yml

```diff
@@ -4,26 +4,25 @@
 name: Angular CI
 
 on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+    push:
+        branches: [main]
+    pull_request:
+        branches: [main]
 
 jobs:
-  build:
+    build:
+        runs-on: ubuntu-latest
 
-    runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node-version: [14.x, 16.x]
```

Comment:
Removed v12 because it isn't supported anymore.
I suppose this is also why the build on azure devops fails.

Ref: https://github.com/angular/angular/blob/main/CHANGELOG.md#core-10

Reply:
I'll start updating the settings to remove 12 as a required validation.

Reply:
Updated, I've also re-created the internal dev-ops the "1DS JavaScript SDK angular plugin", it's now on a different server -- can you please try pushing a change again to trigger the new workflow... As I can't see your branch to manually trigger it from the server and the current one just complains that the build is unavailable.

Reply:
Hmm, the auto triggering of the new workflow doesn't seem to work, but I found a work-around by manually triggering with the  latest hash...
Also as part of updating the node versions, my PR updates to `actions/setup-node@v3` using (for now) 12, 14... I tried adding 16, but Angular v13 fails to build -- so will need to wait until the final v13 releases and pushed into it's own branch to finalize that change *sigh*